frontend/src/api/actions/init.ts

```diff
@@ -97,7 +97,7 @@ export function initFromStage(
     stage: InitStages,
     options = { startAfterStage: false }
 ): ThunkAction<Promise<void>, RootState, void, AnyAction> {
-    const startAfterStage = +!!options.startAfterStage;
+    const startAfterStage = options.startAfterStage ? 1 : 0;
```

Comment:
Thought it would be easier to read without sacrificing the guaranteed type conversion